src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine{
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck is working.");
```

Comment:
The message in the `doWork` method should be more informative. According to the checklist, it should indicate what type of `Machine` is working. Consider changing the message to something like 'Truck started its work.' to make it clear and informative.